src/App.js

```diff
@@ -1,8 +1,76 @@
-function App() {
+import { useState } from "react";
+import sunny from "./sunny.png"
+import rainy from "./rainy.png"
+import semi from "./semi.png"
+import cloudy from "./cloudy.png"
+
+var picture= sunny;
+
+
+function App() { 
+
+
+var APIdata 
+const [data,setData ] = useState({} );
+const [location,setLocation ] = useState("" );
+
+const apiKey = "ffcc3b1d6839acddfd18b10b824cdae1";
+const apiUrl=  "https://api.openweathermap.org/data/2.5/weather?&units=metric&q=";
+
+ const checkWeather = async () => {
+ const response = await fetch(apiUrl + location +'&appid=' + apiKey);
+  APIdata = await response.json();
+ setData(APIdata)
```

Comment:
Here you can add some error handling, you can add try catch block and save error message which you can later display below input field if lets say user enters non existing city. If that doesn't work maybe APIdata returns error message so you can just save that one and display it...